backend/fms_core/models/dataset_file.py

```diff
@@ -1,23 +1,37 @@
 from django.db import models
 import reversion
+from django.core.exceptions import ValidationError
 
 from .tracked_model import TrackedModel
 from .readset import Readset
 
-from ._constants import STANDARD_FILE_PATH_LENGTH, ReleaseStatus, ValidationStatus
+from ._constants import STANDARD_FILE_PATH_LENGTH, ValidationStatus
+from ._utils import add_error as _add_error
 
 @reversion.register()
 class DatasetFile(TrackedModel):
     """ Class to store information about the files associated with their dataset for data deliveries. """
 
     readset = models.ForeignKey(Readset, on_delete=models.PROTECT, help_text="Readset of the file.", related_name="files")
-    file_path = models.CharField(max_length=STANDARD_FILE_PATH_LENGTH, help_text="Path to the dataset file.")
-    release_status = models.IntegerField(choices=ReleaseStatus.choices, default=ReleaseStatus.AVAILABLE, help_text="The release status of the file.")
-    release_status_timestamp = models.DateTimeField(null=True, blank=True, help_text='The last time the release status of the file was changed.')
+    file_path = models.CharField(null=False, max_length=STANDARD_FILE_PATH_LENGTH, help_text="Path to the dataset file.")
+    size = models.IntegerField(null=False, help_text="Size of the dataset file.")
     validation_status = models.IntegerField(choices=ValidationStatus.choices, default=ValidationStatus.AVAILABLE, help_text="The run validation status of the file.")
```

Comment:
Does not really make sense to move the release status and let the validation status on the dataset_file. The validation status and timestamp should be moved either to readset or to the dataset (we are validating at that level anyway ...).